types/craftos/craftos.d.ts

```diff
@@ -140,680 +150,775 @@ declare var _G: LuaTable;
+  function cosh(x: number): number;
+  function deg(x: number): number;
+  function exp(x: number): number;
+  function floor(x: number): number;
+  function fmod(x: number, y: number): number;
+  function frexp(x: number): number;
+  var huge: number;
+  function ldexp(m: number, e: number): number;
+  function log(x: number): number;
+  function log10(x: number): number;
+  function max(x: number, ...args: number[]): number;
+  function min(x: number, ...args: number[]): number;
+  function modf(x: number): number;
+  var pi: number;
+  function pow(x: number, y: number): number;
+  function rad(x: number): number;
+  function random(m?: number, n?: number): number;
+  function randomseed(x: number): number;
+  function sin(x: number): number;
+  function sinh(x: number): number;
+  function sqrt(x: number): number;
+  function tan(x: number): number;
+  function tanh(x: number): number;
 }
 declare namespace multishell {
-    function getFocus(): number;
-    function setFocus(n: number): void;
-    function getTitle(n: number): string | null;
-    function setTitle(n: number, title: string): void;
-    function getCurrent(): number;
-    function launch(env: LuaTable, path: string, ...args: string[]): number;
-    function getCount(): number;
+  function getFocus(): number;
+  function setFocus(n: number): void;
+  function getTitle(n: number): string | null;
+  function setTitle(n: number, title: string): void;
+  function getCurrent(): number;
+  function launch(env: LuaTable, path: string, ...args: string[]): number;
+  function getCount(): number;
 }
 declare type LuaDate = {
-    year: number;
-    month: number;
-    day: number;
-    hour: number;
-    min: number;
-    sec: number;
-    wday: number;
-    yday: number;
-    isdst: boolean;
-}
+  year: number;
+  month: number;
+  day: number;
+  hour: number;
+  min: number;
+  sec: number;
+  wday: number;
+  yday: number;
+  isdst: boolean;
+};
 
 declare namespace os {
-    function version(): string;
-    function getComputerID(): number;
-    function computerID(): number;
-    function getComputerLabel(): string | null;
-    function computerLabel(): string | null;
-    function setComputerLabel(label?: string | null): void;
-    function run(env: LuaTable, path: string, ...args: string[]): boolean;
-    function queueEvent(type: string, ...param: any[]): void;
-    function clock(): number;
-    function startTimer(timeout: number): number;
-    function cancelTimer(id: number): void;
-    function time(mode?: string | null): number;
-    function sleep(timeout: number): void;
-    function day(mode?: string | null): number;
-    function setAlarm(time: number): number;
-    function cancelAlarm(id: number): void;
-    function shutdown(): void;
-    function reboot(): void;
-    function epoch(mode?: string | null): number;
-    function date(format?: string | null, time?: number | null): string | LuaDate;
+  function loadAPI(path: string): boolean;
```

Comment:
`os.[un]loadAPI` were intentionally removed, as they are deprecated, and not necessary in TypeScript considering `import` is built into the language. Please remove them.

Reply:
I was using the CC wiki when I structured this and I was under the impression that it was maintained. I was unaware that this method was deprecated although it would make my life easier. Can you refer me the where you're getting this info?

Reply:
nvm, i see https://tweaked.cc/module/os.html#v:loadAPI